src/Microsoft.DocAsCode.Dfm/DfmRenderer.cs

```diff
@@ -12,10 +13,16 @@ namespace Microsoft.DocAsCode.Dfm
 
     public class DfmRenderer : HtmlRenderer
     {
+        private readonly ImmutableDictionary<string, string> _tokens;
         private static readonly DocfxFlavoredIncHelper _inlineInclusionHelper = new DocfxFlavoredIncHelper();
         private static readonly DocfxFlavoredIncHelper _blockInclusionHelper = new DocfxFlavoredIncHelper();
         private static readonly DfmCodeExtractor _dfmCodeExtractor = new DfmCodeExtractor();
 
+        public DfmRenderer(ImmutableDictionary<string, string> tokens)
```

Comment:
for example, OpDfmRenderer in op plugg-in is child of this class, which not pass tokens as parameters

---
In reply to: [67845126](https://github.com/dotnet/docfx/pull/403#discussion_r67845126) [](ancestors = 67845126)